src/spral/metis4_wrapper.F90

```diff
@@ -156,12 +156,12 @@ subroutine metis_order64(n,ptr,row,perm,invp,flag,stat)
 
     ! Carry out ordering
     metis_opts(1) = 0 ! MeTiS defaults
-    call metis_nodend(n,ptr2,row2,1,metis_opts,invp,perm)
```

Comment:
Nick, don't you have an infinite `loop`if `galahad_metis` calls `galahad_metis`?

Reply:
Do I? I don't see it

Reply:
I checked again and you call `metis_nodend` on the last commit of `multi-precision` so you already fixed and It was just an old commit. I resolve the conversation 🙂